packages/lodestar/src/chain/clock/LocalClock.ts

```diff
@@ -43,6 +44,17 @@ export class LocalClock implements IBeaconClock {
     return getCurrentSlot(this.config, this.genesisTime);
   }
 
+  /**
+   * Maximum currentSlot of peers.
+   * If it's too close to next slot given MAXIMUM_GOSSIP_CLOCK_DISPARITY, return currentSlot + 1.
+   * Otherwise return currentSlot
+   */
+  public get maxPeerCurrentSlot(): Slot {
```

Comment:
I don't get how is this max slot of peers. I guess you meant "wall clock"?

Reply:
yeah it's related to that. When we're too close to next slot, other peers may switched to next slot already and it submits/spread the new block to the network and our validation failed if we use our own `currentSlot`. Since the spec allow a `MAXIMUM_GOSSIP_CLOCK_DISPARITY`, I implement this method accordingly.

The method name may be confusing, any recommendation?

Reply:
maybe networkWallClock or assumedWallClock? Not sure

Reply:
Imo `currentSlotWithGossipDisparity`